dips/dip-158.md

```diff
@@ -1,42 +1,36 @@
 ---
 dip: 158
-title: P2M (eCommerce) Payments Off-Chain Protocol
+title: P2M Charge Payment - Off-Chain Protocol
```

Comment:
This PR mistakenly includes DIP 158: P2M Charge Payment. Let's cut this file.

Reply:
Closed this PR.
Will open a new one after DIP-158 changes is merged to main.